provision/acc_provision/acc_provision.py

```diff
@@ -545,6 +546,22 @@ def config_adjust(args, config, prov_apic, no_random):
     return adj_config
 
 
+def is_valid_mtu(xval):
+    if xval is None:
+        # use default configured on thsi host
+        return True
+
+    xmin = 1280   # for IPv6
```

Comment:
Seems like the default we were are using is 1500:
https://github.com/noironetworks/aci-containers/blob/f1a564c9abdf320eaf3af3654bbd5c02c092ce96/pkg/hostagent/config.go#L177

Should xmin be 1500?

Reply:
No, the user can set it below the default - just not lower than 1280 as IPv6 requires that (IPv4 min is lower, but opflex supports IPv6 so we will go with that min).